src/index.js

```diff
@@ -0,0 +1,130 @@
+// Constants
+const BASE_URL = 'http://127.0.0.1:5000';
+const DEFAULT_CITY = 'Los Angeles';
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset'); 
+const headerCityName = document.getElementById('headerCityName');
+const currentTempButton = document.getElementById('currentTempButton');
+
+// Changes color + landscape by Temperature
+let temp = 60;
+const changeTemp = () => {
+    let landscapeEmojis = '';
+    if (temp >= 80) {
+        tempValue.style.color = 'red';
```

Comment:
This syntax is the JS equivalent of setting an inline style on a tag, which we generally want to avoid. Prefer using the `className` or `classList` attributes to modify the CSS classes associated with an element instead. This provides much more flexibility, as we can change many style properties simply by updating the CSS rules.